src/TesApi.Web/BatchDeletionException.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace TesApi.Web
+{
+    /// <summary>
```

Comment:
Exception class with Exception properties?

Reply:
Would deriving from AggregationException be more appropriate here?